r/tests/testthat/test-chunked-array.R

```diff
@@ -206,7 +206,7 @@ test_that("ChunkedArray supports empty arrays (ARROW-13761)", {
     int8(), int16(), int32(), int64(), uint8(), uint16(), uint32(),
     uint64(), float32(), float64(), timestamp("ns"), binary(),
     large_binary(), fixed_size_binary(32), date32(), date64(),
-    decimal(4, 2), dictionary(), struct(x = int32())
+    decimal(4, 2), decimal128(4, 2), dictionary(), struct(x = int32())
```

Comment:
I think we can get rid of `decimal(...)` here since it's just a wrapper for `decimal128()`, right?

Reply:
I thought I'd keep all the tests for `decimal()` since we haven't deprecated it, but happy to remove them.

Reply:
Done